prow/config.yaml

```diff
@@ -220,6 +220,19 @@ tide:
     istio-ecosystem/authservice: squash
   target_url: https://prow.istio.io/tide
 
+tide:
+  queries:
+  - repos:
+    - istio/operator
+    labels:
+    - review:approved
```

Comment:
You don't want to search for a label named `review:approved` you want to ensure github considers the PR approved, which is `reviewApprovedRequired: true` (aka a sibling to labels)

```yaml
- repos:
  - istio/operator
  labels:
  - "cla: yes"  # do we care about this label?
  missingLabels: # what you have now
  reviewApprovedRequired: true
```

Reply:
And since I know you stay up at night wondering about how to configure tide and prow, here's what you have to work with right now: https://github.com/kubernetes/test-infra/blob/3f92da8679e28b626a91cf5f168ceee014d038f1/prow/config/tide.go#L168-L182